aws/rapids_ec2_mnmg.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.                         security_groups=['sg-name'],
</pre><ol><li>If you want to use an existing security group , definitely add some instructions that &nbsp;it should allows ports 8786, 8787 and all internal traffic and all outbound traffic or point the users to https://cloudprovider.dask.org/en/latest/aws.html#elastic-compute-cloud-ec2 . If you use a security group without the necessary ports open, then the cluster creation step would be fine, but the worker vms wont connect to the scheduler and will sit idle at client.wait_for_workers step. I have ran into this.</li><li>Further, if you do not specify a security group,  and you do not have permissions to create one (which was the case with me), it will fail to create a security group <code>un</code>gracefully. Ran into this as well. </li><li>In some cases, the cluster creation may throw an error that it is out of p3.8xlarge instances in the availability zone (it apparently picks something by default). You may want to create the cluster by passing <code>availability_zone=""</code>    to allow EC2Cluster to select zones agnostically based on availability in case the user is not fixated on having every VM on a single availability zone in a region. </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Thanks! Updated the text.</p><p> </p><p>The availability zone is decided by the AWS_REGION in this case from credentials.</p>

Reply:
I think the AWS_REGION only mentions the region and not the zone.